packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -190,6 +196,7 @@ export const OdieSendMessageButton = () => {
 				>
 					<ResizableTextarea
 						shouldDisableInputField={ isChatBusy || isAttachingFile }
```

Comment:
We can maybe keep the ResizableTextarea without any zendesk logic by passing cantTransferToZendesk to disabled and passing a new placeholder property

Reply:
Good idea!